lib/merge_supporters.rb

```diff
@@ -14,6 +14,13 @@ def self.update_associations(old_supporters, new_supporter, np_id, profile_id)
         .where("supporter_id IN ($ids)", ids: old_supporter_ids).timestamps.execute
     end
 
+    old_supporters.joins(:addresses).each do |supp|
+      supp.addresses.each do |addr|
+        addr.supporter = new_supporter
+        addr.save!
```

Comment:
I don't think we need to merge in all of the addresses. After all, each supporter can only have a single address at this point. Since the user can manually set what they want the address of the new supporter to be, there's no need to update any of the old addresses. We'll need something like this in later parts of this project but not this one.